src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -1,13 +1,12 @@
-import logging
 import random
 from datetime import datetime, timedelta
 from typing import Any, Dict, List, Optional, Set, Tuple
 
 import requests
-from celery.utils.log import get_task_logger
```

Comment:
i couldn't figure out a quick way to integrate this with `loguru`. i can create a follow up ticket if we think this is important functionality